samples/python/tutorial_code/features2D/Homography/perspective_correction.py

```diff
@@ -0,0 +1,62 @@
+import cv2 as cv
+import numpy as np
+import sys
+
+
+def randomColor():
+    color = np.random.randint(0, 255,(1, 3))
+    return color[0].tolist()
+
+def  perspectiveCorrection(img1Path, img2Path ,patternSize ):
+    img1 = cv.imread(cv.samples.findFile(img1Path))
+    img2 = cv.imread(cv.samples.findFile(img2Path))
+
+
+    ret1, corners1 = cv.findChessboardCorners(img1, patternSize)
+    ret2, corners2 = cv.findChessboardCorners(img2, patternSize)
+
+    if not ret1 or not ret2:
+        print("Error, cannot find the chessboard corners in both images.")
+        sys.exit(-1)
+
+
+    H, _ = cv.findHomography(corners1, corners2)
+
+
+    img1_warp = cv.warpPerspective(img1, H, (img1.shape[1], img1.shape[0]))
+
+    img_draw_warp = cv.hconcat([img2, img1_warp])
+    cv.imshow("Desired chessboard view / Warped source chessboard view", img_draw_warp )
+
+    corners1 = corners1.tolist()
+    corners1 = [a[0] for a in corners1]
+
+    img_draw_matches = cv.hconcat([img1, img2])
+    for i in range(len(corners1)):
+        pt1 = np.array([corners1[i][0], corners1[i][1], 1])
+        pt1 = pt1.reshape(3, 1)
+        pt2 = np.dot(H, pt1)
+        pt2 = pt2/pt2[2]
+        end = (int(img1.shape[1] + pt2[0]), int(pt2[1]))
+        cv.line(img_draw_matches, tuple([int(j) for j in corners1[i]]), end, randomColor() , 2)
+
+    cv.imshow("Draw matches", img_draw_matches)
+    cv.waitKey(0)
+
+def main():
+    import argparse
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-I1', "--image1", help="Path to the first image", default="left02.jpg")
+    parser.add_argument('-I2', "--image2", help="Path to the second image", default="left01.jpg")
```

Comment:
Seems a bit confusing that `I1` is `left02.jpg` but `I2` is `left01.jpg`.

Reply:
In the [original c++ sample ](https://github.com/opencv/opencv/blob/master/samples/cpp/tutorial_code/features2D/Homography/perspective_correction.cpp) it was like that's why I used I1 for lefto2.jpg and I2 for left01.jpg

Reply:
Got it, thanks!

Reply:
I would suggest to keep the same description than in the C++ sample:
> path to the source chessboard image
> path to the desired chessboard image

Since it is a perspective correction sample, maybe use instead `src` and `dst` arguments? But if so, make it consistent between the C++ and Python code.